client/src/components/Export.vue

```diff
@@ -0,0 +1,125 @@
+<script>
+import { getExportUrl } from '@/common/api';
+import {
+  ImageSequenceType,
+  VideoType,
+} from '@/constants';
+
+const MediaTypeMap = {
+  [ImageSequenceType]: 'image sequence',
+  [VideoType]: 'video',
+};
```

Comment:
should we make the display names for the types universal within the program so if they are used more than once we don't have conflicting terms?